Description: Remove ';' and some shell snippets in .c files that make GCC 4.3 fail
Bug-Debian: http://bugs.debian.org/420983
Author: Stefan Fritsch <sf@debian.org>
Last-Update: 2011-11-01

Index: sam2p/gensi.hpp
===================================================================
--- sam2p.orig/gensi.hpp	2011-11-01 18:42:54.855587237 +0100
+++ sam2p/gensi.hpp	2011-11-01 18:50:50.158343490 +0100
@@ -253,7 +253,7 @@
       if (len!=0) block(beg,len,data);
       block(0,0,data);
     }
-    inline virtual void first_sub(Sub &sub) const { sub.beg=beg; sub.len=len; };
+    inline virtual void first_sub(Sub &sub) const { sub.beg=beg; sub.len=len; }
     inline virtual void next_sub(Sub &sub) const { sub.len=0; }
     inline char const*getCString() const { return beg; }
     inline char const*operator()() const { return beg; }
Index: sam2p/ps_tiny.c
===================================================================
--- sam2p.orig/ps_tiny.c	2011-11-01 18:42:54.859587260 +0100
+++ sam2p/ps_tiny.c	2011-11-01 18:50:50.158343490 +0100
@@ -1,10 +1,3 @@
-#define DUMMY \
-set -ex; \
-gcc -O3 -s -DNDEBUG=1 -DNO_CONFIG=1 -ansi -pedantic -Wunused \
-  -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline \
-  -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes \
-  -Wmissing-declarations ps_tiny.c -o ps_tiny; \
-exit
 /* ps_tiny.c -- read .pin files, write short PostScript code
  * by pts@fazekas.hu at Sat Aug 24 12:46:14 CEST 2002
  * -- Sat Aug 24 18:23:08 CEST 2002
Index: sam2p/pts_defl.c
===================================================================
--- sam2p.orig/pts_defl.c	2011-11-01 18:42:54.863587283 +0100
+++ sam2p/pts_defl.c	2011-11-01 18:50:50.162343510 +0100
@@ -1,10 +1,3 @@
-#define DUMMY \
-set -ex; \
-"${CC:-gcc}" -DNDEBUG=1 -DPTS_DEFL_MAIN -O3 -ansi \
-  -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline \
-  -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes \
-  -Wmissing-declarations pts_defl.c -o flateenc; \
-exit
 /*
  * pts_defl.c -- C source file ZIP compression ripped from linux-2.6.8.1
  * by pts@fazekas.hu at Tue Jan 18 15:19:06 CET 2005
Index: sam2p/pts_lzw.c
===================================================================
--- sam2p.orig/pts_lzw.c	2011-11-01 18:37:22.401667519 +0100
+++ sam2p/pts_lzw.c	2011-11-01 18:50:50.162343510 +0100
@@ -1,10 +1,3 @@
-#define DUMMY \
-set -ex; \
-g++ -DNDEBUG=1 -O3 -ansi -pedantic \
-  -Wall -W -Wstrict-prototypes -Wtraditional -Wnested-externs -Winline \
-  -Wpointer-arith -Wbad-function-cast -Wcast-qual -Wmissing-prototypes \
-  -Wmissing-declarations pts_lzw.c -c; \
-exit
 /* pts_lzw.c -- a real, effective implementation of PostScript
  *   LanguageLevel2 and PDF /LZWEncode and /LZWDecode filters (same as the LZW
  *   compression used in TIFF raster image files)
